kem/hybrid/hybrid.go

```diff
@@ -56,18 +56,21 @@ var kyber512X kem.Scheme = &scheme{
 	"Kyber512-X25519",
 	kyber512.Scheme(),
 	hpke.KEM_X25519_HKDF_SHA256.Scheme(),
+	0x01fd,
```

Comment:
These are temporary and we might not want to include them, yet.

Reply:
from where these numbers came from, please add a link to a reference.